### ISSUE TITLE
gradlew build error with checkstyleMain?

### ISSUE THREAD

**Issue by gweeyc** [nus-tic4002-AY2122s2/forum#12]
When I run gradlew build, I encounter this error reported:
![image](https://user-images.githubusercontent.com/43993134/152981740-c73cd043-cc57-47a2-9c3d-74f0a3ed1eb0.png)

When I run gradlew check, the error is similar: -
![image](https://user-images.githubusercontent.com/43993134/152982516-d1b965d6-3974-41da-8484-9e3e794bd7ea.png)

I notice after adding the JavaFx GUI, it always failed at checkstyleMain? How can this issue be resolved successfully?
Thank you!

**Comment by damithc** [comment]
@gweeyc what Java versions have you installed in your comp? Do you have Java 16 by any chance?

**Comment by gweeyc** [comment]
Hi Prof Damith,
                            I install only Java 11: -
***@***.***

                         The ip Project Settings: -
***@***.***

Thanks
Gwee Yeu Chai

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by gweeyc** [comment]
Hi Prof Damith,
I am using Amazon Coretto Java 11 and Gradle version 7.3.3 only. 
When I run gradlew clean check, and look into the reports/checkstyle/main.xml file I get this error also: -

![image](https://user-images.githubusercontent.com/43993134/153012236-3439b69e-9948-4225-8670-d9cbaaf2ce1b.png)

**Comment by damithc** [comment]
@gweeyc push your latest code to a branch and post the link here. I'll get someone to talk a look.

**Comment by gweeyc** [comment]
HI Prof Damith,
I found that the module-info.java is causing the issue most probably, think there is sth I'm not understanding about module config in Java:
```
module ip.duke.gui {
    requires javafx.fxml;
    requires javafx.graphics;
    requires javafx.controls;

    opens ip.duke.gui to javafx.controls, javafx.graphics, javafx.fxml;
}
```

I deleted this file, and configure with the standard config, I can now run checkstyleMain ! Just that when I start the JavaFx, there is this warning msg:
![image](https://user-images.githubusercontent.com/43993134/153088773-1a76cd54-4765-4c81-b82c-4d79ee51fa2d.png)

Shall I leave this warning alone or is there a solution to it? The ChecksytleMain can produce the main.html report already, no problem.

Thank you!
